### ISSUE TITLE
Cannot find gcov or GCOV with Bazel 0.17.1

### ISSUE THREAD

**Issue by cushon** [bazelbuild/bazel#6180]
Building Android Studio with Bazel 0.17.1 fails with the following error:

```
ERROR: in target '//external:cc_toolchain': no such package '@local_config_cc//': Traceback (most recent call last):
        File "external/bazel_tools/tools/cpp/cc_configure.bzl", line 56
                configure_unix_toolchain(repository_ctx, cpu_value, overriden...)
        File "external/bazel_tools/tools/cpp/unix_cc_configure.bzl", line 486, in configure_unix_toolchain
                _find_generic(repository_ctx, "gcov", "GCOV", overri...)
        File "external/bazel_tools/tools/cpp/unix_cc_configure.bzl", line 465, in _find_generic
                auto_configure_fail(msg)
        File "external/bazel_tools/tools/cpp/lib_cc_configure.bzl", line 109, in auto_configure_fail
                fail(("\n%sAuto-Configuration Error:%...)))

Auto-Configuration Error: Cannot find gcov or GCOV; either correct your path or set the GCOV environment variable
```

**Comment by cushon** [comment]
This was a regression introduced by a95a44270010ec3643b73c0296ef0f76e447362f and fixed in 1d956c707e1c843896ac58a341c335c9c149073d (which didn't make the baseline for 0.17.1).

**Comment by cushon** [comment]
This will be fixed in 0.17.2 (https://github.com/bazelbuild/bazel/issues/6164#issuecomment-422841842) and 0.18x.
